README.md

```diff
@@ -10,3 +10,25 @@ The purpose of this project is to learn
 - [Bootstrap](http://getbootstrap.com/)
 - [Babel](https://babeljs.io/)
 - [Firebase Hacker News API](https://hacker-news.firebaseio.com/)
+
+## Getting started
+
+Make sure you have [node.js](https://nodejs.org) installed
+
+Install the hn-clone package and its dependencies.
+
+```bash
+npm install
+```
+
+Run the build script
+
+```bash
+npm run build
+# or alternatively the file watching build script
+npm run watch
+```
+
+Serve the site using Python's SimpleHTTPServer at http://localhost:8000
```

Comment:
@Krylik I will look at using that. I went with Python because it should be included in most operating systems and SimpleHTTPServer is a built in library.